packages/web-components/src/player-component/player.class.ts

```diff
@@ -154,6 +154,7 @@ export class PlayerWrapper {
     private _driftCorrectionTimer: number | null = null;
     private wallclock_event: shaka_player.PlayerEvents.FakeEvent | undefined;
     private _errorHandler: shakaErrorHandler;
+    private _rtspPlayback: boolean = false;
```

Comment:
I wonder if instead of adding this (which seems to mirror the existing 'rtsp-playback' attribute, which you could use instead - of course I don't know how to, myself), could we just replace all instances of this._rtspPlayback with this.player.getAssetUri().startsWith('ws')?

Reply:
However, be wary of nulls. So maybe this.player?.getAssetUri()?.startsWith('ws').